internal/app/telescopes/api/handlers.go

```diff
@@ -131,6 +131,49 @@ func (r *RouteHandler) recommendClusterScaleOut() gin.HandlerFunc {
 	}
 }
 
+// swagger:route POST /recommender/ recommend recommendClustersSetup
+//
+// Provides a recommended set of node pools on a given provider in a specific region.
+//
+//     Consumes:
+//     - application/json
+//
+//     Produces:
+//     - application/json
+//
+//     Schemes: http
+//
+//     Security:
+//
+//     Responses:
+//       200: RecommendationResponse
+func (r *RouteHandler) recommendClustersSetup() gin.HandlerFunc {
```

Comment:
let's rename this to recommendMultiCluster and the "old one" to recommendCluster